Packs/BitSight/Integrations/BitSightForSecurityPerformanceManagement/BitSightForSecurityPerformanceManagement.py

```diff
@@ -61,9 +63,14 @@
     'spf': 'SPF',
     'ssl certificates': 'ssl_certificates',
     'ssl configurations': 'ssl_configurations',
-    'unsolicited communications': 'unsolicited_comm'
-}
+    'unsolicited communications': 'unsolicited_comm',
+    'web application security': 'web_appsec',
+    'dmarc': 'dmarc'
 
+}
+PACK_VERSION = get_pack_version(pack_name='Bitsight') or '1.1.22'
```

Comment:
Using the `1.1.22` for `PACK_VERSION` might return an outdated version number after the next package update. Consider avoiding hard-coded values to ensure the version always matches the actual package version.

Reply:
Hi @YairGlik, The reason for putting the constant for the version number is that sometimes `get_pack_version` doesn't provide the response for the `fetch-incidents`. So for the safer side, we have put it with the `OR` condition.